scripts/execwhacker.py

```diff
@@ -20,6 +20,7 @@
 
 import ahocorasick
 from bcc import BPF
+from lookup_container import ContainerNotFound, ContainerType, get_container_id, lookup_container_details_crictl, lookup_container_details_docker
```

Comment:
This is ok for now, but I think we should move this into a package (than a bunch of scripts) soon.